Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -36,6 +38,20 @@ struct DashboardView: View {
                         } label: {
                             Image(systemName: "trash")
                         }
+                        
+                        Button {
+                            selectedMaintenanceEvent = event
+                            isShowingEditView = true
+                        } label: {
+                            VStack {
+                                Text("Edit")
+                                Image(systemName: "pencil")
+                            }
+                        }
+                    }
+                    .sheet(isPresented: $isShowingEditView) {
```

Comment:
Just a note the sheet can be moved on the same level of modifiers as the main stuff (it doesn't have to specifically be here